packages/flutter/lib/src/material/bottom_sheet_theme.dart

```diff
@@ -44,6 +45,11 @@ class BottomSheetThemeData extends Diagnosticable {
   /// If null, [BottomSheet] defaults to 0.0.
   final double elevation;
 
+  /// Value for [BottomSheet.elevation], if specified.
```

Comment:
Specify that this is only when the Bottom sheet is presented as a persisten bottom sheet

Reply:
Oh, I guess specify that this is for modal bottom sheets only now

Reply:
Done